src/SamplesApp/SamplesApp.UnitTests.Shared/Controls/UnitTest/UnitTestsControl.cs

```diff
@@ -225,16 +239,19 @@ private void ReportTestClass(TypeInfo testClass)
 				Windows.UI.Core.CoreDispatcherPriority.Normal,
 				() =>
 				{
-					var testResultBlock = new TextBlock()
+					if (!IsRunningOnCI)
```

Comment:
Can the condition be placed at the `RunAsync` call itself? Otherwise we will be calling `RunAsync` to run an "effectively" empty lambda.

Reply:
It can :) Thanks